src/MediatR/Unit.cs

```diff
@@ -29,10 +29,7 @@ public struct Unit : IEquatable<Unit>, IComparable<Unit>, IComparable
         ///  - Zero: This object is equal to <paramref name="other" />.
         ///  - Greater than zero: This object is greater than <paramref name="other" />.
         /// </returns>
-        public int CompareTo(Unit other)
-        {
-            return 0;
-        }
+        public int CompareTo(Unit other) => 0;

```

Comment:
Is there a change to editorconfig you can do to make this preferred?

Reply:
Probably, but it'd be R#-only